workspace/apps/pidp/src/app/features/access/pages/ms-teams-privacy-officer/ms-teams-privacy-officer.page.ts

```diff
@@ -107,15 +107,6 @@ export class MsTeamsPrivacyOfficerPage
     );
   }
 
-  public onBack(): void {
```

Comment:
This button maybe worth keeping. The MS Teams Privacy Officer enrolment has 3 pages that the user needs to read before they can submit the enrolment. We could eliminate the `Home` button on the first page, which would be the same button we are eliminating from every other enrolment